Sources/UI/Notifications/Models/Notification.swift

```diff
@@ -51,7 +51,7 @@ extension Notification {
 }
 
 extension Notification: AttributeRepresentable {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.